app/build.gradle

```diff
@@ -24,6 +28,15 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
+    androidTestImplementation 'com.android.support.test:rules:1.0.1'
+    androidTestImplementation 'com.android.support.test.espresso:espresso-intents:3.0.1'
     implementation 'com.android.support:recyclerview-v7:26.1.0'
     implementation 'com.google.code.gson:gson:2.8.0'
+    androidTestUtil 'com.android.support.test:orchestrator:1.0.2'
+    androidTestCompile 'com.android.support.test.espresso:espresso-contrib:2.2.2', {
```

Comment:
Should probably use androidTestImplementation instead of androidTestCompile

compile is now deprecated in gradle